package.json

```diff
@@ -20,13 +18,26 @@
   "scripts": {
     "build": "rm -rf lib && babel src --out-dir lib --ignore *.spec.js",
     "test": "npm run test:lint && npm run test:unit && npm run test:integration",
-    "test:lint": "eslint .",
+    "test:lint": "standard | snazzy",
+    "test:lint:fix": "standard --fix",
```

Comment:
shouldn't these new `scripts` be added to the README.md? Or they are supposed to be used only as part of `test` internally